widget/accordion.go

```diff
@@ -129,20 +129,26 @@ type accordionRenderer struct {
 	widget.BaseRenderer
 	container *Accordion
 	headers   []*Button
+	dividers  []*canvas.Rectangle
 }
 
 func (r *accordionRenderer) Layout(size fyne.Size) {
+	divHeight := 1
 	x := 0
 	y := 0
 	for i, ai := range r.container.Items {
 		if i != 0 {
-			y += theme.Padding()
+			y += theme.Padding() * 2
+
+			div := r.dividers[i-1]
+			div.Move(fyne.NewPos(x, y-1-theme.Padding())) // middle of padding gap between items
+			div.Resize(fyne.NewSize(size.Width, divHeight))
 		}
 		h := r.headers[i]
 		h.Move(fyne.NewPos(x, y))
 		min := h.MinSize().Height
 		h.Resize(fyne.NewSize(size.Width, min))
-		y += min
+		y += min + divHeight
```

Comment:
This even adds `divHeight` to the `padding` space after the divider (which has then `padding-1` before and `padding+divHeight` after). This line should be reverted, especially with the proposed changes from above.

```suggestion
		y += min
```

Reply:
Good catch, fixed in response to above comment